data/common.yaml

```
@@ -5,3 +5,5 @@ profile_hardening::remove_setuid_setgid::files:
   - "/usr/bin/nvidia-modprobe"
   - "/usr/libexec/Xorg.wrap"
   - "/usr/sbin/login_duo"
+
+profile_hardening::sysctl::settings: []
```

Nitpicky but could you set it to this to show it's a hash?
```suggestion
profile_hardening::sysctl::settings: {}
```

---

Good catch! I missed that, but I've opened a new pull request: https://github.com/ncsa/puppet-profile_hardening/pull/3